evaldo/builtins_spreadsheet.go

```diff
@@ -1366,7 +1397,7 @@ func ColNames(ps *env.ProgramState, from env.Object, fnName string) ([]string, e
 			switch k := k1.(type) {
 			case env.String:
 				cols[i] = k.Value
-			case env.Word:
+			case env.Tagword:
```

Comment:
I think this should be the correct behavior for column names, right?

Reply:
I haven't looked at concrete code yet, but in general tagwords (this is sort of a temp name, called lit-words in  rebol) evaluate to words by evaluator, but in a block (which doesn't get evaluated by default they remain tagwords). It's still a little open question if tagwords in blocks make sense, or should they just be words then. I like that they emphasise their inert nature with left quote ... like indexed strings, but it's not totally consistent with all other uses maybe. This will remain open question for now.